docs/core/resilience/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Introduction to resiliency in .NET
+description: Learn about resiliency in .NET and how to build a resilience pipeline.
+author: IEvangelist
+ms.author: dapine
+ms.date: 09/27/2023
+---
+
+# Introduction to resiliency in .NET
+
+Resiliency is the ability of an app to recover from failures and continue to function. In the context of .NET programming, resiliency is achieved by designing apps that can handle failures gracefully and recover quickly. To help build resilient apps in .NET, the following two packages are available on NuGet:
+
+- [Microsoft.Extensions.Resilience](https://www.nuget.org/packages/Microsoft.Extensions.Resilience): This NuGet package provides mechanisms to harden apps against transient failures.
+- [Microsoft.Extensions.Http.Resilience](https://www.nuget.org/packages/Microsoft.Extensions.Http.Resilience): This NuGet package provides resiliency mechanisms specifically for the <xref:System.Net.Http.HttpClient>.
+
+These two NuGet packages are built on top of _Polly_, which is a very popular open-source project. Polly is a .NET resilience and transient-fault-handling library that allows developers to express strategies such as Retry, Circuit Breaker, Timeout, Bulkhead Isolation, Rate-limiting and Fallback in a fluent and thread-safe manner. For more information, see [Polly](https://github.com/App-vNext/Polly).
```

Comment:
```suggestion
These two NuGet packages are built on top of [Polly](https://github.com/dotnet/docs/pull/37270), which is a very popular open-source project. Polly is a .NET resilience and transient fault-handling library that allows developers to express strategies such as Retry, Circuit Breaker, Timeout, Bulkhead Isolation, Rate-limiting and Fallback in a fluent and thread-safe manner.
```

Reply:
Another, as you talk about things like Retry, or Circuit Breaker, consider adding links to this really nice Azure design pattern pages which describe the various patterns: https://learn.microsoft.com/en-us/azure/architecture/patterns/